src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -49,6 +49,7 @@ public WorkspaceInfo()
                 double cx = 0;
                 double cy = 0;
                 double zoom = 1.0;
+                double scaleFactor = 1.0;
```

Comment:
there is a UI associated with this which webUI might need - in addition I am not sure if we need to scale the tessellated geometry after it is returned from ASM or if LibG is doing this scaling to tessellated geometry @aparajit-pratap can you fill us in here?